infra/src/test/kotlin/fi/fta/geoviite/infra/geometry/VerticalGeometryListingTest.kt

```diff
@@ -70,7 +70,7 @@ class VerticalGeometryListingTest() {
 
         val angleFraction = angleFractionBetweenPoints(point1, point2)
         assertNotNull(angleFraction)
-        assertEquals(angleFraction, 1.0, 0.0001)
+        assertEquals(angleFraction, 1.0, 0.000001)
```

Comment:
Näitä raja-arvoja muutettu vastaamaan enemmän sitä mitä rajapinta oikeasti palauttaa, aiemmat oli vedetty hatusta